ui/desktop/electron-app/src/index.js

```diff
@@ -54,6 +56,33 @@ protocol.registerSchemesAsPrivileged([
 // correctly inherit the path variable in production
 fixPath();
 
+// Setup logger
+log.initialize();
+log.transports.console.level = false;
+log.transports.file.format =
+  '[{y}-{m}-{d} {h}:{i}:{s}.{ms}{z}] [{level}] {text}';
+log.transports.file.fileName = 'desktop-client.log';
+log.transports.file.archiveLogFn = (file) => {
+  file = file.toString();
+  const info = path.parse(file);
+
+  // This renames old files to the format of `desktop-client-(iso string date).log`
+  // TODO: Should we compress or delete old log files after a certain time?
```

Comment:
Not exactly sure how we should handle log rotation currently. The default logic is it rotates to a `file.old.log` so it will constantly delete old files by just renaming them (i.e. there's a `main.log` and a `main.old.log` and with default log size of 1MB, it basically keeps at most 2MB of the most recent logs).

I didn't want it to lose older logs but now I'm also not deleting any old logs, do we think it's necessary to have some sort of cleanup before we release? Or for now would it be better to keep the default logic and just only have it keep the two most recent log files?

Reply:
Since we removed the rotation logic, will we now delete the old logs and only show the two recent logs that fit into our maxsize?

Reply:
Yup exactly

Reply:
It'd be the same log file

Reply:
Thanks for clarification